src/splitInteger.test.js

```diff
@@ -4,18 +4,35 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
+  const result = splitInteger(17, 4);
 
+  expect(result).toEqual([4, 4, 4, 5]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
+  const result = splitInteger(8, 1);
 
+  expect(result).toEqual([8]);
 });
 
 test('should sort parts ascending if they are not equal', () => {
+  const result = splitInteger(32, 6);
+  const sortedResult = [...result].sort((a, b) => a - b);
 
+  expect(result).toEqual(sortedResult);
 });
 
 test('should add zeros if value < numberOfParts', () => {
+  const result = splitInteger(1, 2);
 
+  expect(result).toEqual([0, 1]);
+});
```

Comment:
This test may not be necessary as it's not specified in the task description that the function should add zeros if the value is less than the number of parts. The function should only split the integer into the specified number of parts, and if the value is less than the number of parts, it should return an array with the value and the rest as zeros.